aws/logs_monitoring/parsing.py

```diff
@@ -527,6 +528,47 @@ def cwevent_handler(event, metadata):
     yield data
 
 
+# Extract findings from Security Hub logs
```

Comment:
I think you should use docstring in this case like this https://github.com/DataDog/datadog-serverless-functions/blob/3d962e943bffffd658cb289e09c1357273e170b2/aws/logs_monitoring/lambda_function.py#L79-L81.